Header header
float32 set_speed
float32 max_speed
float32 min_speed
float32 accel
float32 deccel
uint8 step_pin
uint8 dir_pin
int64 steps_per_rev
float32 lead
float64 length
string unit

